osu.Desktop/OsuGameDesktop.cs

```diff
@@ -102,35 +100,21 @@ protected override UpdateManager CreateUpdateManager()
             if (!string.IsNullOrEmpty(packageManaged))
                 return new NoActionUpdateManager();
 
-            switch (RuntimeInfo.OS)
-            {
-                case RuntimeInfo.Platform.Windows:
-                    Debug.Assert(OperatingSystem.IsWindows());
-
-                    return new SquirrelUpdateManager();
-
-                default:
-                    return new SimpleUpdateManager();
-            }
+            return new VelopackUpdateManager();
         }
 
         public override bool RestartAppWhenExited()
         {
-            switch (RuntimeInfo.OS)
+            try
             {
-                case RuntimeInfo.Platform.Windows:
-                    Debug.Assert(OperatingSystem.IsWindows());
-
-                    // Of note, this is an async method in squirrel that adds an arbitrary delay before returning
-                    // likely to ensure the external process is in a good state.
-                    //
-                    // We're not waiting on that here, but the outro playing before the actual exit should be enough
-                    // to cover this.
-                    Squirrel.UpdateManager.RestartAppWhenExited().FireAndForget();
-                    return true;
+                Velopack.UpdateExe.Start();
+                return true;
+            }
+            catch (Exception e)
+            {
+                Logger.Error(e, "Failed to restart application");
```

Comment:
This messaging is wrong. It's not failing to restart anything; it's failing to start the updater.

Reply:
> it's failing to start the updater.

Which, in turn, results in a failure to restart the application. I can make it more specific, but it's not entirely wrong to say that restarting the application failed (since that *is* the purpose of the method). Would you rather I completely reword it to "Failed to launch restarter process", or clarify with "Failed to start restarter process. A restart will not be preformed"

Reply:
Don't change anything; see the other comment above which removes this completely.